lstchain/scripts/tests/test_lstchain_scripts.py

```diff
@@ -66,7 +66,7 @@ def simulated_dl1ab(temp_dir_simulated_files, simulated_dl1_file):
     )
     return output_file
 
-
+@pytest.fixture(scope="session")
 def test_add_source_dependent_parameters(simulated_dl1_file):
```

Comment:
A test is not a fixture. With adding this, you just made that the test is not actually run.

A fixture is an input for a test. Let me check why the test is failing.

I think the test is actually good and needed and we need to figure out why it failed

Reply:
sorry, the fixture is wrong, the test was running without problems

Reply:
great!

Reply:
The problem came from:
> In the last master merging, conflicts were not properly solved and two test_add_source_dependent_parameters functions were left.

Reply:
Yes, sorry, that was me doing a not trivial merge using the web interface. Should have checked locally.